ui/src/app/model/Summary.ts

```diff
@@ -0,0 +1,8 @@
+export default interface ISummary {
```

Comment:
Owing to consistency rename all files in model folder to start with lower case

Reply:
Added to backlog as it may require changes in the backend also. 
Backlog no - 51